examples/emberjs/todomvc/app/components/todo-item.gjs

```diff
@@ -0,0 +1,68 @@
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { service } from '@ember/service';
+import { isBlank } from '@ember/utils';
+
+export default class TodoItem extends Component {
+  <template>
+    <li class="{{if @todo.completed 'completed'}} {{if this.editing 'editing'}}">
```

Comment:
Unless I am misreading this, shouldn't `this.editing` be tracked?

Reply:
you are correct -- that fixes the double-click not working bug that @a13o found. thank you